lib/src/parse/stylesheet.dart

```diff
@@ -2854,7 +2877,22 @@ abstract class StylesheetParser extends Parser {
           buffer.addInterpolation(interpolatedString().asInterpolation());
           wroteNewline = false;
 
-        case $slash when scanner.peekChar(1) == $asterisk:
+        case $slash:
+          switch (scanner.peekChar(1)) {
+            case $asterisk:
+              buffer.write(rawText(loudComment));
+              wroteNewline = false;
+
+            case $slash when silentComments:
+              silentComment();
+              wroteNewline = false;
+
+            case _:
+              buffer.writeCharCode(scanner.readChar());
+              wroteNewline = false;
+          }
+
+        case $slash when silentComments && scanner.peekChar(1) == $slash:
           buffer.write(rawText(loudComment));
```

Comment:
should this really parse a loud comment when finding `//` ? Or is this dead code because of going through the previous case all the time ?

Reply:
You're right that it's incorrect and also that it's dead code. I'll send out a PR to remove it.